src/common/interfaces.ts

```diff
@@ -97,3 +95,25 @@ export interface Tree {
   kennzeich?: string | null;
   caretaker?: string | null;
 }
+
+// Legend shared
+export interface StyledProps {
+  active?: boolean;
+}
+
+export interface FlexColumnProps {
+  isLast?: boolean;
```

Comment:
Same for all of the inserted interfaces in this file. This file is not a place to simply transfer all the types into. It should only contain the minimum and only business logic relate `interfaces` like `TreeType` or `UserType`.